ext/js/data/options-util.js

```diff
@@ -997,4 +998,36 @@ class OptionsUtil {
         }
         return options;
     }
+
+    async _updateVersion21(options) {
+        await this._applyAnkiFieldTemplatesPatch(options, '/data/templates/anki-field-templates-upgrade-v21.handlebars');
+
+        let customTemplates = false;
+        for (const {options: profileOptions} of options.profiles) {
+            if (profileOptions.anki.fieldTemplates !== null) {
+                customTemplates = true;
+            }
+        }
+
+        if (customTemplates && isObject(chrome.storage)) {
```

Comment:
Forgotten `chrome.storage` check? (I take it you are trying to eliminate these now that all supported browsers are supposed to have `chrome.storage`)

Reply:
i added the check here to not fail the test harness.